GraphicsView/include/CGAL/Qt/camera_impl.h

```diff
@@ -885,11 +885,8 @@ CGAL_INLINE_FUNCTION
 Vec Camera::pointUnderPixel(const QPoint &pixel, bool &found) const {
   float depth = 2.0;
   // Qt uses upper corner for its origin while GL uses the lower corner.
-  if(parent())
-  {
     dynamic_cast<QOpenGLFunctions*>(parent())->glReadPixels(pixel.x(), screenHeight() - 1 - pixel.y(), 1, 1,
                                                             GL_DEPTH_COMPONENT, GL_FLOAT, &depth);
-  }
```

Comment:
If looks like it is `dynamic_cast<QOpenGLFunctions*>(parent())` that should be tested against `nullptr`.